### PR TITLE
[EDR Workflows] Correct bugs in Execute action API docs

### PR DESCRIPTION
**Pull request by kevinlog**
Updates the Execute API docs based on user feedback.

- Corrects bug in one of the API parameters where a string was listed, but it should be an integer in seconds
- Adds a note to inform the user that the document upload functionality is rate limited, so they should batch bulk Endpoint calls.

### CODE REVIEW

**Review by natasha-moore-elastic — COMMENTED** [review]
Thanks for making these updates @kevinlog! LGTM, I've left a couple of questions and minor suggestions.

File: docs/management/api/execute-api.asciidoc [hunk 1]
```diff
@@ -22,13 +22,14 @@ A JSON object with these fields:
 |`case_ids` |Array (String) |The IDs of cases where the action taken will be logged. |No
 |`comment` |String |Attach a comment to this action's log. The comment text will appear in associated cases. |No
 |`parameters.command` |String |A shell command to run on the host. The command must be supported by `bash` for Linux and macOS hosts, and `cmd.exe` for Windows. |Yes
-|`parameters.timeout` |String |How long the host should wait for the command to complete. Use `h` for hours, `m` for minutes, `s` for seconds (for example, `2s` is two seconds). If no timeout is specified, it defaults to four hours. |No
+|`parameters.timeout` |Integer |How long the host should wait for the command to complete in seconds.  |No
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/management/api/execute-api.asciidoc:25]: Alternate streamlined wording:
```suggestion
|`parameters.timeout` |Integer |The duration, in seconds, that the host waits for the command to complete.  |No
```

**Review by joepeeples — COMMENTED** [review]
@natasha-moore-elastic I'm finishing this review as "Comment" since I think our new branch protections will cancel any approval once you commit the open edit suggestions. Please ping me again when the edits are merged and I can give the necessary approval! ✅

**Comment by natasha-moore-elastic** [comment]
> @natasha-moore-elastic I'm finishing this review as "Comment" since I think our new branch protections will cancel any approval once you commit the open edit suggestions, but everything LGTM. Please ping me again when the edits are merged and I can give the necessary approval! ✅

Hey @joepeeples, did you mean to tag me here? The PR was opened by @kevinlog.

File: docs/management/api/execute-api.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (dasansol92) [reply docs/management/api/execute-api.asciidoc:25]: I guess it shouldn't, @paul-tavares @ashokaditya can you confirm that?

**Comment by dasansol92** [comment]
>For my own understanding, what's the behavior if this field is not specified?

@natasha-moore-elastic it defaults to 4 hours.
cc: @ashokaditya @paul-tavares

File: docs/management/api/execute-api.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashokaditya) [reply docs/management/api/execute-api.asciidoc:25]: No the changed sentences for `timeout` do not affect the overall description of the parameter.

[the remark this answers is not in the document]
Comment (ashokaditya) [reply docs/management/api/execute-api.asciidoc:25]: @nastasha-solomon if the `timeout` parameter is not specified at action input it defaults to a 4 hour (14400 seconds) duration

**Comment by natasha-moore-elastic** [comment]
> > For my own understanding, what's the behavior if this field is not specified?
> 
> @natasha-moore-elastic it defaults to 4 hours. cc: @ashokaditya @paul-tavares

Thanks @dasansol92, I will then add back this sentence, which was originally part of the description:

> If no timeout is specified, it defaults to four hours.

**Pull request merged**